docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,83 +1,145 @@
 
-Submit the following indexing request to add a single document to the
-`books` index.
-The request automatically creates the index.
+Before you begin, you need to have a running {es} cluster.
+The fastest way to get started is with a <<run-elasticsearch-locally,local development environment>>.
+Refer to <<elasticsearch-intro-deploy,Run {es}>> for other deployment options.
 
 ////
 [source,console]
 ----
 PUT books
+PUT my-explicit-mappings-books
 ----
 // TESTSETUP
 
 [source,console]
 --------------------------------------------------
 DELETE books
+DELETE my-explicit-mappings-books
 --------------------------------------------------
 // TEARDOWN
 
 ////
 
+[discrete]
+[[getting-started-index-creation]]
+=== Step 1: Create an index
+
+Create a new index named `books`:
+
+[source,console]
+----
+PUT /books
+----
+// TEST[skip: index already setup]
+
+A `200` response indicates the index was created successfully.
+
+.Expand to see example response
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "acknowledged": true,
+  "shards_acknowledged": true,
+  "index": "books"
+}
+----
+// TEST[skip: index already setup]
+===============
+
+[discrete]
+[[getting-started-add-documents]]
+=== Step 2: Add data to your index
+
+[TIP]
+====
+This tutorial uses {es} APIs, but there are many other ways to
+<<es-ingestion-overview,add data to {es}>>.
+====
+
+You add data to {es} as JSON objects called documents.
+{es} stores these
+documents in searchable indices.
+
+[discrete]
+[[getting-started-add-single-document]]
+==== Add a single document
+
+Submit the following indexing request to add a single document to the
+`books` index.
+
+[TIP]
+====
+If the index didn't already exist, this request would automatically create it.
+====
+
 [source,console]
 ----
 POST books/_doc
 {"name": "Snow Crash", "author": "Neal Stephenson", "release_date": "1992-06-01", "page_count": 470}
 ----
-// TEST[s/_doc/_doc?refresh=wait_for/]
+// TEST[continued]
 
-The response includes metadata that {es} generates for the document including a unique `_id` for the document within the index.
+The response includes metadata that {es} generates for the document, including a unique `_id` for the document within the index.
 
-.Expand to see example response
+.Expand to see example response with annotations
 [%collapsible]
 ===============
 [source,console-result]
 ----
 {
-  "_index": "books",
-  "_id": "O0lG2IsBaSa7VYx_rEia",
-  "_version": 1,
-  "result": "created",
-  "_shards": {
-    "total": 2,
-    "successful": 2,
-    "failed": 0
+  "_index": "books", <1>
+  "_id": "O0lG2IsBaSa7VYx_rEia", <2>
+  "_version": 1, <3>
+  "result": "created", <4>
+  "_shards": { <5>
+    "total": 2, <6>
+    "successful": 2, <7>
+    "failed": 0 <8>
   },
-  "_seq_no": 0,
-  "_primary_term": 1
+  "_seq_no": 0, <9>
+  "_primary_term": 1 <10>
 }
 ----
-// TEST[skip:TODO]
+// TEST[s/O0lG2IsBaSa7VYx_rEia/*/]
+<1> The `_index` field indicates the index the document was added to.
```

Comment:
this is a lot. do we have this anywhere else? perhaps we defer to the metadata docs for this level of detail.

Reply:
it's optional tho so I'm wondering if we leave it in because if you don't know you don't know and it's helpful

Reply:
it is a lot but don't see resource where everything is laid out together

see https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html


let me know how strongly feel about this one

Reply:
are we possibly missing metadata fields in the linked reference then?  I agree with keeping annotations on the non-metadata fields